### PR TITLE
Marvel/postsapp add debt views

### PR DESCRIPTION
**Pull request by Reekofgeek**

### CODE REVIEW

File: SDM/SDM/settings/base.py [hunk 1]
```diff
@@ -116,9 +117,8 @@
 
 STATIC_URL = 'static/'
 
-STATICFILES_DIRS = [
-    BASE_DIR / 'static'
-]
+STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static') ]
```
Comment (jpphilips) [inline SDM/SDM/settings/base.py:120]: What's the difference?

File: SDM/posts/models.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from tkinter import CASCADE
```
Comment (jpphilips) [inline SDM/posts/models.py:1]: What's this for?

File: SDM/SDM/settings/base.py (hunk 1 above)
Comment (Reekofgeek) [reply SDM/SDM/settings/base.py:120]: They do the same but its advisable to use os.path.join incase of static issues in future when deploying.

File: SDM/posts/models.py (hunk 2 above)
Comment (Reekofgeek) [reply SDM/posts/models.py:1]: Don't know how that got there must have been a wrongly typed code that brought that up sha.

**Review by jpphilips — APPROVED** [review]

**Pull request closed without merging**